testing-endpoints.md

```diff
@@ -18,20 +18,60 @@ curl -X GET http://localhost:8080/api/test/did-debug
 ```
 
 
+#### `api/test/generate-did`
+
+Returns a Long-Form did made from input public keys in jwk format
+
+The body is a JSON object (note, this API requires a content-type header) with the following fields:
+1. `recoveryPublicJwk` - **required** `ES256` public signing key jwk for did recovery operations
+2. `updatePublicJwk` - **required** `ES256` public signing key jwk for did update operations
+3. `signingPublicJwk` - **optional** `ES256` public signing key jwk for verifying assertions 
+4. `encryptionPublicJwk` - **optional** public encryption key jwk using the `"alg": "ECDH-ES"` and `"enc": "A256GCM"` encryption algorithm
+5. `domains` - **optional** array of string URI hostnames. Will be added to the did's `LinkedDomains` by an `add-services` action. 
+
+**Note**: The `signingPublicJwk` and `encryptionPublicJwk` fields are optional since 
+not all entities represented by DIDs may need to sign or encrypt documents.
+
+For exmample, a `verifier` or `issuer` entity that does not request SIOP Responses to be encrypted MAY only need to have a signing key (for signing SIOP requests), and not an encryption key.
```

Comment:
```suggestion
For example, a `verifier` or `issuer` entity that does not request SIOP Responses to be encrypted MAY only need to have a signing key (for signing SIOP requests), and not an encryption key.
```

However, I think this doc line even further suggests that we should automatically create signing keys.